utils/iotop/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2017 Bjørn Mork <bjorn@mork.no>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iotop
+PKG_VERSION:=0.6
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Bjørn Mork <bjorn@mork.no>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://guichaz.free.fr/iotop/files/
```

Comment:
Huh?  Works for me, and is the canonical upstream source of this package

bjorn@canardo:~$ host guichaz.free.fr
guichaz.free.fr is an alias for perso130-g5.free.fr.
perso130-g5.free.fr has address 212.27.63.130
perso130-g5.free.fr has IPv6 address 2a01:e0c:1:1597::130
bjorn@canardo:~$ ping guichaz.free.fr
PING perso130-g5.free.fr (212.27.63.130) 56(84) bytes of data.
64 bytes from perso130-g5.free.fr (212.27.63.130): icmp_seq=1 ttl=56 time=67.0 ms
64 bytes from perso130-g5.free.fr (212.27.63.130): icmp_seq=2 ttl=56 time=67.0 ms
^C
--- perso130-g5.free.fr ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1000ms
rtt min/avg/max/mdev = 67.022/67.036/67.050/0.014 ms
bjorn@canardo:~$ ping6 guichaz.free.fr
PING guichaz.free.fr(2a01:e0c:1:1597::130) 56 data bytes
64 bytes from 2a01:e0c:1:1597::130: icmp_seq=1 ttl=55 time=55.4 ms
64 bytes from 2a01:e0c:1:1597::130: icmp_seq=2 ttl=55 time=55.7 ms
^C
--- guichaz.free.fr ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1001ms
rtt min/avg/max/mdev = 55.422/55.567/55.712/0.145 ms